lib/RedisChannel.cpp

```diff
@@ -6,9 +6,12 @@
 
 #include "swss/logger.h"
 #include "swss/select.h"
+#include "swss/events.h"
 
 using namespace sairedis;
 
+event_handle_t g_events_handle;
```

Comment:
According to offline sync, I suggest not necessary use global or class level event publisher, we can just use a publisher on stack to save resource, because we only need send failed event.

Reply:
please dont use global objects